core/variant/variant.h

```diff
@@ -447,6 +451,11 @@ class Variant {
 
 	operator IPAddress() const;
 
+	template <typename T>
+	_FORCE_INLINE_ operator TypedArray<T>() const { return operator Array(); }
```

Comment:
I kind of asked this before, but it's always relevant: Do we _need_ this to be an implicit conversion, or could we be fine making it an explicit one?